src/Orm/Zed/KernelApp/Persistence/SpyAppConfigQuery.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Orm\Zed\KernelApp\Persistence;
+
+use Spryker\Zed\KernelApp\Persistence\Propel\AbstractSpyAppConfigQuery as BaseSpyAppConfigQuery;
+
+/**
+ * Skeleton subclass for performing query and update operations on the 'spy_app_config' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements. This class will only be generated as
+ * long as it does not already exist in the output directory.
+ */
+class SpyAppConfigQuery extends BaseSpyAppConfigQuery
```

Comment:
Do you need this?

Reply:
it comes with spryker-feature/spryker-core update.

Reply:
b2c have it, marketplace does not have

Reply:
maybe this feature is not a part of marketplace